amundsen_application/static/js/components/SearchPage/index.tsx

```diff
@@ -151,6 +152,7 @@ export class SearchPage extends React.Component<SearchPageProps, SearchPageState
     };
     return (
         <div className="search-list-container">
+          <BookmarkList />
```

Comment:
If I had to choose, this could go after popular tables. On a smaller screen I don't even see any hint that Popular Tables still exists if I have 5+ bookmarks...at least the other way around "My Bookmarks" title at least fits into view. If Product is on board, after this PR how do you feel about taking some time to think about a good "HomePage" layout?

Reply:
Resolving as we spoke offline about plans for a "HomePage", layout will be decided as part of that PR.